### ISSUE TITLE
checkFeatures does not catch unsupported features when using comma separator

### ISSUE THREAD

**Issue by jlowcs** [yowainwright/es-check#388]
When running `pnpm es-check es2022 --checkFeatures`.

This gets flagged as `Unsupported features detected: ArrayToReversed`

```js
42;[].toReversed()
```

But this does not:

```js
42,[].toReversed()
```

We got really confused when we enabled sourcemaps in the Sentry Webpack plugin as it was transforming a comma into a semicolon. All of a sudden, errors started appearing when running `es-check`.
